docs/reference/overrides/number-field-root.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "NumberFieldRoot",
+  "props": {
+    "format": {
+      "type": "Intl.NumberFormatOptions"
+    },
+    "onValueChange": {
+      "type": "(value, event) => void"
+    }
+  }
+}
```

Comment:
Point of interest: this is how generated type definitions are overridden when they are no good